sql/ts/tests/node.play.ts

```diff
@@ -1,9 +1,12 @@
-import { test, type Page, expect } from "@playwright/test";
+import { test, expect } from "@playwright/test";
 import { tests } from "./tests";
-import { createSkdb } from "skdb";
+import { createSkdb, type SKDB } from "skdb";
 
 function run(t, asWorker: boolean) {
   test(t.name, async () => {
+    if (t.timeout) {
```

Comment:
I took care of this and another test in #182. I spent time ssh'd in to circleci understanding resource usage. I'll land it in a minute.